cookbooks/bcpc-hadoop/recipes/hive_hcatalog.rb

```diff
@@ -53,15 +53,6 @@
   user 'hdfs'
 end
 
-bash 'create-hive-scratch' do
-  code <<-EOH
-  hdfs dfs -mkdir -p #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chmod 1777 #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chown hive:hdfs #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  EOH
-  user 'hdfs'
-end
```

Comment:
Allan, can you check if Hive will create the directory if the user has perms and it doesn't exist? That way we can be lazy and blow this off (and nice not to run `O (users)` more HDFS actions).